src/components/App/App.js

```diff
@@ -12,6 +13,10 @@ import NotFound from "pages/NotFound/NotFound";
 const baseURL = process.env.REACT_APP_BASE_APP_URL;
 
 function App() {
+  if (process.env.NODE_ENV === "production") {
+    ReactGA.initialize("UA-1018242-68");
+    ReactGA.pageview(window.location.pathname + window.location.search);
```

Comment:
We probably also want the hash so a more resilient approach might be to do...Unless we don't, in which case ignore this :-)
```suggestion
    ReactGA.pageview(window.location.href.replace(window.location.origin, ''));
```